src/App.js

```diff
@@ -4,12 +4,15 @@ import { Provider } from "react-redux";
 import { BrowserRouter, Routes, Route } from "react-router-dom";
 
 import Home from "pages/Home";
+import Modal from "components/global/Modal";
```

Comment:
move this line next to ModalProvider in order to group related components together